include/sound/sof/dai-intel.h

```diff
@@ -82,6 +82,7 @@ struct sof_ipc_dai_ssp_params {
 struct sof_ipc_dai_hda_params {
 	struct sof_ipc_hdr hdr;
 	uint32_t link_dma_ch;
+	uint32_t reserved[4];
```

Comment:
why be greedy, can this be increased to future proof?

Also there's no explanation on why this is needed at all, so will trigger comments upstream.

Reply:
This really doesn't belong here...